h5p-bildetema/src/components/SearchPage/SearchPage.tsx

```diff
@@ -36,12 +37,11 @@ const SearchParamKeys = {
 const SearchPage = (): JSX.Element => {
   const languages = useLanguagesWithTranslatedLabels();
   const langCode = useCurrentLanguageCode();
+  const siteLanguage = useSiteLanguage();
```

Comment:
You could also add  viewLanguages={viewLanguages} as an array here

Reply:
Yes, will do so if we allow several view languages 👍